Markdownyfied README.md

habi opened the pull request:
As part of http://24pullrequests.com/

habi committed:
Markdownified README and added links

As part of http://24pullrequests.com/

habi committed:
Fixed markdown issues.

chfoo commented:
Hi, thanks. Some concerns though:
1. Any reason why you changed the headers to atx-style? I'm just curious as to why not leave it the same style as the original author had used.
2. The 3 lines describing the files look a bit clumped together. A unnumbered list would work better.
3. I see you tried to convert the steps into a numbered list with code blocks but then went with inline code. Since the code line in step 2 is fairly long, it should be kept as a block. Code blocks should work in a list with blank lines before & after and indented using 8 spaces.

If you make the changes for concerns 2 & 3, it should be ok for merge.

habi committed:
Update README based on suggestions

@chfoo had some suggestions on my pull request (https://github.com/ArchiveTeam/warrior-preseed/pull/1#issuecomment-66351440).
Trying to implement them.

habi committed:
Corrected links

habi commented:
> Any reason why you changed the headers to atx-style? I'm just curious as to why not leave it the same style as the original author had used.
> Purely personal preference and now reverted.
> The 3 lines describing the files look a bit clumped together. A unnumbered list would work better.
> That looks better, indeed.
> I also added direct links to the files.
> 
> I see you tried to convert the steps into a numbered list with code blocks but then went with inline code. Since the code line in step 2 is fairly long, it should be kept as a block. Code blocks should work in a list with blank lines before & after and indented using 8 spaces.
> I think that the inline code looks nicer.
> The long code line is now reverted back to a block.

chfoo commented:
Ah, sorry, I should be more clear. 

For example (with periods added for clarity):

```
1. Blah Blah

........This is a code block
........which may span lines.

2. This line contains `inline code`. The case is obvious here.
3. Blah Blah

....`This is also inline code. It just happens to be on an individual line. This is not a code block.`

```

So I meant it was ok to keep the keep the formatting of the steps (ie, ok to keep the code in the same line of the step) except for Step 2 which should be using a code block.

Sorry for all the fuss, I should have explained it better.

habi committed:
Another edit to format the steps

habi committed:
Get rid of the period. Period.
